Test/git-issues/git-issue-1604e.dfy.expect

```diff
@@ -0,0 +1,15 @@
+git-issue-1604e.dfy(65,37): Error: Could not prove that the bound variable 'go' satisfies the (non runtime testable) type GhostOrdinalCell after the range, where it's assumed to be of type Cell based on inferred bounds.
```

Comment:
I'm changing almost everything according to your suggestions, except 1) I do include the type of the collection element in the error message and 2) the reason in in the last one, because that would touch other pieces of error reporting. The original error I catch is "a ghost predicate is allowed only in specification contexts (consider declaring the predicate with 'predicate method')", and I just wrap it with a "because".

Here is the final rendering:
```
git-issue-1604e.dfy(65,37): Error: Could not prove that the range constrains the bound variable 'go' to be of type GhostOrdinalCell, and since GhostOrdinalCell is a ghost subset type, no runtime type filter can be added on the collection whose elements have the type Cell.
git-issue-1604e.dfy(24,24): [Related location] The subset type GhostOrdinalCelltype GhostOrdinalCell is ghost because it depends on the ghost subset type GhostNaturalCell
git-issue-1604e.dfy(13,35): [Related location] GhostNaturalCell is ghost because a call to a ghost predicate is allowed only in specification contexts (consider declaring the predicate with 'predicate method')
```